src/main/java/core/basesyntax/service/impl/FruitListImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.FruitList;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitListImpl implements FruitList {
```

Comment:
The class name 'FruitListImpl' is not very descriptive of its responsibilities. Consider renaming it to reflect its purpose more accurately, such as 'FruitTransactionListServiceImpl' if it's meant to provide a list of fruit transactions.